### PR TITLE
Update image size

### PR DESCRIPTION
**Pull request by imchairmanm**
Smaller image to address #389

### CODE REVIEW

**Comment by nilubava** [comment]
@imchairmanm Is this ready to be merged?

**Comment by imchairmanm** [comment]
> @imchairmanm Is this ready to be merged?

Still waiting on a review from someone.  Can merge after that.

**Review by alex-emerich — APPROVED** [review]

**Pull request merged**
